src/biokbase/narrative/contents/narrativeio.py

```diff
@@ -211,6 +211,7 @@ def write_narrative(self, obj_ref, nb, cur_user):
                 meta[u'job_ids'][u'apps'] = list()
             if 'job_usage' not in meta[u'job_ids']:
                 meta[u'job_ids'][u'job_usage'] = {u'queue_time': 0, u'run_time': 0}
+            meta[u'is_temporary'] = 'false'
```

Comment:
OK, so according to bill in this case, as opposed to the narrative service, Untitled is a fine name for a non-temporary narrative. Saving a narrative here implies that 1) the user has changed the name or 2) the user has run an app. Hence the narrative is no longer empty and not temporary.

Effectively, once the narrative service creates an empty narrative, the front end cannot save temporary narratives.